addons/mail/static/src/components/thread_needaction_preview/thread_needaction_preview.scss

```diff
@@ -0,0 +1,104 @@
+// ------------------------------------------------------------------
+// Layout
+// ------------------------------------------------------------------
+
+.o_ThreadNeedactionPreview {
+    @include o-mail-notification-list-item-layout();
+
+    &:hover .o_ThreadNeedactionPreview_markAsRead {
+        opacity: 1;
+    }
```

Comment:
cannot be mixin'ed due to hover prefix?

Reply:
Cannot be mixin'ed due to prefix of specific BEM classname being dependent on component name (`o_ThreadNeedactionPreview_markAsRead` => `ThreadNeedactionPreview`).

We have a task-2258605 to fix this issue.